pulpcore/tests/functional/api/using_plugin/test_repo_versions.py

```diff
@@ -1211,8 +1211,7 @@ def test_all(self):
 
         self.assertEqual(ctx.exception.status, 400)
 
-        # No repository version exists.
-        self.assertEqual(self.repo_ver_api.list().count, 0)
+        initial_rv_count = self.repo_ver_api.list().count
```

Comment:
```suggestion
        initial_rv_count = self.repo_ver_api.list(limit=0).count
```
Not sure if it really works, but if we only care about the total number there is no need to ask for any detail.